requirements.txt

```diff
@@ -1,38 +1,55 @@
 # For outdated module report see https://requires.io/github/OpenTreeMap/otm-core/requirements/?branch=develop
 # Changelog URL shown below if not linked on above page
 
-boto==2.42.0                             # http://docs.pythonboto.org/en/latest/releasenotes/v2.39.0.html
-# https://docs.djangoproject.com/en/1.10/releases/#id2
-Django==1.11.3  # rq.filter: >=1.11,<1.12
-django-apptemplates==1.2
+amqp==1.4.9
+anyjson==0.3.3
+billiard==3.3.0.23
```

Comment:
On the develop branch I can export a CSV but on this branch I get these errors. Not sure if it is actually a problem with billiard.

```
[2017-08-10 12:52:03,957: INFO/MainProcess] Received task: exporter.tasks.async_csv_export[242bf504-3aaa-46d9-a915-fd6096a9b4f5]
[2017-08-10 12:52:03,965: DEBUG/MainProcess] Task accepted: exporter.tasks.async_csv_export[242bf504-3aaa-46d9-a915-fd6096a9b4f5] pid:11152
[2017-08-10 12:52:07,646: ERROR/MainProcess] Task exporter.tasks.async_csv_export[242bf504-3aaa-46d9-a915-fd6096a9b4f5] raised unexpected: WorkerLostError('Worker exited prematurely: signal 11 (SIGSEGV).',)
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/billiard/pool.py", line 1175, in mark_as_worker_lost
    human_status(exitcode)),
WorkerLostError: Worker exited prematurely: signal 11 (SIGSEGV).
```